### ISSUE TITLE
Minimize Downtime During Backup

### ISSUE THREAD

**Issue by mcintyred** [pivotalservices/cfops#42]
Would it be possible to have an option to control when the cloud controller is stopped and started? At the moment the CC is down during the whole backup process, which is painful when the foundation is being used by people in different geographies as there is never a good time for downtime.

I think it should be OK to
  - backup blobs
  - stop the cc
  - back up the cc, uaa etc databases (should be quick since these DBs should not get too big)
  - start the cc
  - backup newly added blobs (should almost never happen)

Does that look right?

**Comment by mcintyred** [comment]
I've now figured out that this is nonsense - blobs are backed up from the ops-manager tile.

**Comment by xchapter7x** [comment]
Agreed, there is room for optimization in this process, that will avoid prolonged downtime.
The Story is in tracker we will scope it and try to get it in the backlog.
